docs/index.md

```diff
@@ -45,7 +53,7 @@ If necessary, provision all OSG hosts that are in your site plan and that do not
 -   [Security information for OSG signed RPMs](release/signing)
 -   [Using Yum and RPM](release/yum-basics)
 -   [Install the OSG repositories](common/yum)
--   [OSG Software release series](release/release_series) - look here to upgrade from OSG 3.1 to OSG 3.2 or from OSG 3.2 to OSG 3.3
+-   [OSG Software release series](release/release_series) - look here to upgrade between releases 3.1 to 3.4.
```

Comment:
This sounds like there are only 3.1 -> 3.4 instructions. How about `..between releases 3.1, 3.2, or 3.3 to 3.4`?